crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1156,24 +1156,30 @@ impl World {
         result
     }
 
-    /// Sends an [Event](crate::event::Event).
+    /// Sends an [`Event`](crate::event::Event).
     #[inline]
     pub fn send_event<E: crate::event::Event>(&mut self, event: E) {
+        self.send_batch(std::iter::once(event));
+    }
+
+    /// Sends the default value of the [`Event`](crate::event::Event) of type `E`.
+    #[inline]
+    pub fn send_default_event<E: crate::event::Event + Default>(&mut self) {
+        self.send_batch(std::iter::once(E::default()));
+    }
+
+    /// Sends a batch of [`Event`](crate::event::Event)s from an iterator.
+    #[inline]
+    pub fn send_batch<E: crate::event::Event>(&mut self, events: impl Iterator<Item = E>) {
```

Comment:
I'm worried that one just reading `send_batch` won't understand it's referring to events. Maybe `send_event_batch` could be better.